src/frontend/src/optimizer/plan_node/logical_insert.rs

```diff
@@ -209,7 +209,21 @@ impl ColPrunable for LogicalInsert {
     }
 }
 
-impl ExprRewritable for LogicalInsert {}
+impl ExprRewritable for LogicalInsert {
```

Comment:
Should we also implement it for `Update`?

Reply:
👀

Reply:
It's already implemented🤣
https://github.com/risingwavelabs/risingwave/blob/b7430b1768881da6404c9ae5cde9c0cda6de6c51/src/frontend/src/optimizer/plan_node/logical_update.rs#L92

Reply:
Cool